src/components/docs.css

```diff
@@ -32,14 +32,14 @@
 }
 
 .docs-section h2 {
-  color: var(--orange); /* 🔥 section headers */
+  color: var(--primary); /* 🔥 section headers */
```

Comment:
This is a good change. You don't have to for this PR. But, removing this comment will help as well.